addons/website_event/controllers/main.py

```diff
@@ -275,3 +275,16 @@ def registration_confirm(self, event, **post):
             'attendees': attendees,
             'event': event,
         })
+
+    @http.route([
+        "/event/registration_badge_designer/<int:event_id>"
+    ], type='http', auth="user", website=True)
+    def view_designer(self, event_id, **post):
+        dummy, action = request.registry.get('ir.model.data').get_object_reference(
+            request.cr, request.uid, 'event', 'action_event_view')
+        values = {
+            'event_obj': request.registry.get('event.event').browse(
```

Comment:
simply call it `event` (change it in the template too) to avoid confusion with the old convention for `event_obj = self.pool.get('event.event')`